pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/util/proxy/jackson/ObjectMapperSingleton.java

```diff
@@ -4,28 +4,38 @@
 import java.util.Map;
 import java.util.Objects;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
 import com.fasterxml.jackson.core.type.TypeReference;
 import com.fasterxml.jackson.databind.DeserializationFeature;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import org.apache.commons.lang3.StringUtils;
+import org.broadinstitute.dsm.exception.DsmInternalError;
 
 public class ObjectMapperSingleton {
 
+    private static final ObjectMapper objectMapperInstance = new ObjectMapper()
+            .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
+            .setSerializationInclusion(JsonInclude.Include.NON_NULL);
+
+    private static final ObjectMapper nonNullObjectMapper = new ObjectMapper()
+            .configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false)
+            .setSerializationInclusion(JsonInclude.Include.NON_NULL);
```

Comment:
I think we might want to use `ALWAYS`, since there are a number of `serializeNulls` calls in DSM code that is using gson.

Reply:
Ah, now I get it.  I misunderstood how `includeNullValues` and was looking too broadly at existing route transforms in gson, which aren't relevant to these changes.  Sorry!